src/org/ses/android/soap/tasks/AgregarHuellaTask.java

```diff
@@ -0,0 +1,67 @@
+package org.ses.android.soap.tasks;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import org.ksoap2.SoapEnvelope;
+import org.ksoap2.serialization.SoapObject;
+import org.ksoap2.serialization.SoapPrimitive;
+import org.ksoap2.serialization.SoapSerializationEnvelope;
+import org.ksoap2.transport.HttpTransportSE;
+
+
+
+/**
+ * Created by franciscorivera on 1/8/16.
+ */
+public class AgregarHuellaTask extends AsyncTask<Byte, Void, String> {
+
+    private String CodigoPaciente;
+
+    public  AgregarHuellaTask(String CodigoPaciente)
+    {
+        this.CodigoPaciente = CodigoPaciente;
+    }
+
+    @Override
+    protected String doInBackground(Byte ... params)
+    {
+        String resul = "";
+
+        final String NAMESPACE = StringConexion.conexion;
+        final String URL=NAMESPACE+"WSSEIS/WSParticipante.asmx";
```

Comment:
We use `WSSEIS/WSParticipante.asmx` a lot... can we make that a constant somewhere?